### PR TITLE
updating on-http README w/ link to readthedocs

### PR DESCRIPTION
**Pull request by BillyAbildgaard**
added link to readthedocs and explanation showing you can substitute api/1.1 for api/2.0 unless otherwise specified.

@RackHD/corecommitters @keedya @srinia6

### CODE REVIEW

**Comment by BillyAbildgaard** [comment]
should not fail because this is only a readme update...

test this please

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 __`on-http` is the HTTP server for RackHD__
 
-_Copyright 2015, EMC, Inc._
+_Copyright 2016, EMC, Inc._
```
Comment (yyscamper) [inline README.md]: I don't think the copyright should be changed. I remembered the Copyright 2015 means the copyright 2015 to now.

**Comment by yyscamper** [comment]
:+1: after revert the copyright change.

File: README.md (hunk 1 above)
Comment (heckj) [inline README.md]: A more appropriate update would be:

`_Copyright 2015-2016, EMC, Inc._`

**Comment by BillyAbildgaard** [comment]
@yyscamper @heckj fixed the copyright

**Comment by heckj** [comment]
👍

**Comment by yyscamper** [comment]
:+1:

**Pull request merged**